src/main/java/com/jcabi/aspects/aj/MethodLogger.java

```diff
@@ -194,6 +194,7 @@ private Object wrap(final ProceedingJoinPoint point, final Method method,
             new MethodLogger.Marker(point, annotation);
         this.running.add(marker);
         int level = annotation.value();
+        final int exceptionLevel = this.decideExceptionLevel(annotation);
```

Comment:
@fandfisf we don't generally use compound method variable names, let's just leave this as `level` :)